src/makielayout/layoutables/time_axis.jl

```diff
@@ -1,32 +1,42 @@
 using GLMakie, Dates, Makie
 using Unitful
 
-
 struct TimeTicks
     time_unit
-    time_range
     tickformatter
 end
-const UnitfulTimes = (
-    typeof(1.0*u"d"), typeof(1.0*u"hr"), typeof(1.0*u"minute"),
-    typeof(1.0*u"s"), typeof(1.0*u"ms"), typeof(1.0*u"μs"), typeof(1.0*u"ns"))
 
-TimeUnits = Union{
-    Number,  # No time units
-    Nothing, # nothing yet
-    UnitfulTimes...}
+TimeUnits = (:yr, :wk, :d, :hr, :minute, :s, :ms, :μs, :ns, :fs)
```

Comment:
This should go ns (10⁻⁹), ps (10⁻¹²), fs (10⁻¹⁵)

Reply:
Yeah, I know, see: https://github.com/JuliaPlots/Makie.jl/blob/20734db18e1f7e2efb89ebae2ec0b373120ffa09/src/makielayout/layoutables/time_axis.jl#L219
This was a commit in the middle of refactoring the code, just to save the progress ;)